mp2-v1/tests/common/values_extraction.rs

```diff
@@ -62,19 +67,39 @@ impl TestContext {
     pub(crate) async fn prove_mapping_values_extraction(
         &self,
         contract_address: &Address,
+        chain_id: u64,
         slot: u8,
+        evm_word: u32,
+        length: usize,
         mapping_keys: Vec<MappingKey>,
     ) -> Vec<u8> {
-        let slot = slot as usize;
-
         let first_mapping_key = mapping_keys[0].clone();
         let storage_slot_number = mapping_keys.len();
 
         // Initialize the test trie.
         let mut trie = TestStorageTrie::new();
         info!("mapping mpt proving: Initialized the test storage trie");
 
+        // Compute the column identifier. It's only one column for simple mapping values.
+        let column_identifier = F::from_canonical_u64(identifier_for_mapping_key_column(
+            slot,
+            contract_address,
+            chain_id,
+            vec![],
+        ));
+        // Compute the table metadata information.
```

Comment:
For mappings of simple variables, only the value column should be placed in `table_info`, the key column is handled separately in the circuit